src/kindlemint/agents/market_research_agent.py

```
@@ -0,0 +1,702 @@
+            strengths.append(
+                f"Strong review count ({book_data['review_count']} reviews)"
+            )
+
+        # Good BSR
+        bsr_info = book_data.get("bsr_info", {})
+        if bsr_info.get("overall_rank"):
+            try:
+                rank = int(bsr_info["overall_rank"])
+                if rank <= 100000:
+                    strengths.append(f"Strong BSR (#{rank:,})")
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)